conseil-common/src/main/scala/tech/cryptonomic/conseil/common/ethereum/rpc/EthereumRpcCommands.scala

```diff
@@ -0,0 +1,86 @@
+package tech.cryptonomic.conseil.common.ethereum.rpc
+
+import io.circe.{Encoder, Json}
+import io.circe.syntax._
+
+import tech.cryptonomic.conseil.common.rpc.RpcClient.RpcRequest
+
+/**
+  * Ethereum JSON-RPC api methods according to the specification at https://eth.wiki/json-rpc/API
+  * These are selected methods necessary for the Lorre to work.
+  */
+object EthereumRpcCommands {
+
+  /**
+    * Sealed trait to keep the list of the RPC methods only in this file.
+    */
+  sealed trait EthereumRpcMethod
+
+  /**
+    * `eth_blockNumber` Ethereum JSON-RPC api method.
+    * Returns the number of most recent block.
+    */
+  object EthBlockNumber extends EthereumRpcMethod {
+    val rpcMethod = "eth_blockNumber"
+    def request = RpcRequest("2.0", rpcMethod, Nil, "bn")
+  }
+
+  /**
+    * `eth_getBlockByNumber` Ethereum JSON-RPC api method.
+    * If verbosity is true it returns the full transaction objects,
+    * if false only the hashes of the transactions.
+    */
+  object EthGetBlockByNumber extends EthereumRpcMethod {
+    val rpcMethod = "eth_getBlockByNumber"
+    case class Params(number: String, verbosity: Boolean)
+    def request(number: String) = RpcRequest("2.0", rpcMethod, Params(number, false), s"egbbn_$number")
+
+    implicit val encodeParams: Encoder[Params] = new Encoder[Params] {
```

Comment:
pro-tip: since java8+ you can make use of functional interfaces to reduce the boilerplate for Single-method-instances as a lambda, i.e.

```scala
implicit val encodeParams: Encoder[Params] = (params: Params) =>
  Json.arr(
    Json.fromString(params.number),
    Json.fromBoolean(params.verbosity)
  )
```

Reply:
I did it according to Circe doc, your code actually looks better :)